src/main/java/appeng/client/gui/implementations/GuiPatternTerm.java

```diff
@@ -122,6 +133,17 @@ public void drawFG( final int offsetX, final int offsetY, final int mouseX, fina
 			this.tabProcessButton.visible = false;
 		}
 
+		if( this.container.substitute )
+		{
+			this.substitutionsEnabledBtn.visible = true;
+			this.substitutionsDisabledBtn.visible = false;
+		}
+		else
+		{
+			this.substitutionsEnabledBtn.visible = false;
+			this.substitutionsDisabledBtn.visible = true;
+		}
+
```

Comment:
Good idea. The whole gui could see some improvments. But simply not enough time currently to also refactor this.